insecure/fixtures.go

```diff
@@ -40,9 +41,9 @@ func EgressMessageFixture(t *testing.T, codec network.Codec, protocol Protocol,
 	// encodes event to create payload
 	payload, err := codec.Encode(content)
 	require.NoError(t, err)
+	eventID := flow.HashToID(hash.DefaultHasher.ComputeHash(payload))
```

Comment:
does this need to match the value [calculated](https://github.com/onflow/flow-go/blob/master/network/p2p/network.go#L528-L528) by the network layer?

Reply:
Although it does not seem to match the networking layer `EventId`, it is more coherent to use the `EventId` [method](https://github.com/onflow/flow-go/blob/master/network/p2p/network.go#L528-L528).

Reply:
https://github.com/onflow/flow-go/pull/3580/commits/3c84a08a7331610b83cc9981f90f9c24acd326ed